docs/store/slate.md

```diff
@@ -0,0 +1,28 @@
+---
+title: 'Slate'
+description: 'Slate is an open source storage system built on top of Filecoin.'
+breadcrumb: ''
+---
+
+# {{ $frontmatter.title }}
+
+{{ $frontmatter.description }} It is the first open source file storage application designed to encourage collaboration and research across a distributed network. It is a first step towards enabling a thriving network for data storage and transactions powered by IPFS, Filecoin and Textile that is open and usable for everyone.
```

Comment:
```suggestion
{{ $frontmatter.description }} It is the first open source file storage application designed to encourage collaboration and research across a distributed network. The creation of Slate is a major step towards enabling a thriving network for data storage and transactions powered by IPFS, Filecoin, and Textile that is open and usable for everyone.
```

Reply:
It is a first step. I did not write this, it comes from their README, I don't see rationale to change it into a "major" step.